src/py/esse/functionals.py

```diff
@@ -0,0 +1,42 @@
+import json
+from string import Template
+import os
+from esse.utils import read_json_file, dump_json_file
+
+DIR = os.path.dirname(__file__)
+UNIT_PATH = os.path.join(DIR, "data/schema/models_directory/pb/qm/dft")
+PROTO_FILE = os.path.join(UNIT_PATH, "dft_unit_functionals_proto.json")
```

Comment:
In my experience, it's usually better to avoid any shortened names and spell them out, even for minor things.

`UNIT_PATH` -> `UNIT_FUNCTIONALS_PATH`
`PROTO_FILE` -> `PROTOTYPE_FILENAME`
`PROP_TEMPLATE` -> `SCHEMA_WITH_PROPERTIES_TEMPLATE`